pep-0604.rst

```diff
@@ -8,6 +8,7 @@ Type: Standards Track
 Content-Type: text/x-rst
 Created: 28-Aug-2019
 Python-Version: 3.10
+Discussions-To: typing-sig@python.org
```

Comment:
The CI failed because the headers are out of order: `Discussions-To` should follow `BDFL-Delegate`:

https://github.com/python/peps/blob/82a3d1f3dfd3ee293d722131e4bb7d4de4d551e2/pep-0001.txt#L512-L532

Reply:
Thank you!